<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="ATMECSINLT-085" failures="1" tests="1" name="Test validateCHROMEcom.atmecs.sample.testscripts.TestVerifyPhpTravels" time="45.462" errors="0" timestamp="04 Nov 2019 16:36:26 GMT">
  <testcase classname="com.atmecs.sample.testscripts.TestVerifyPhpTravels" name="verifyPhpTravels" time="36.99">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.atmecs.sample.logreports.LogReporter.logReportMessage(LogReporter.java:24)
at com.atmecs.sample.helper.ValidaterHelper.assertValidater(ValidaterHelper.java:158)
at com.atmecs.sample.testscripts.TestVerifyPhpTravels.verifyPhpTravels(TestVerifyPhpTravels.java:39)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:835)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- verifyPhpTravels -->
</testsuite> <!-- Test validateCHROMEcom.atmecs.sample.testscripts.TestVerifyPhpTravels -->
